.allow_skipping_tests

```diff
@@ -47,6 +47,8 @@ helpers/all_casa_admins/casa_orgs_helper.rb
 helpers/contact_types_helper.rb
 helpers/date_helper.rb
 helpers/template_helper.rb
+helpers/api_base_helper.rb
```

Comment:
Please write tests for code instead of ignoring them

Reply:
got it